### PR TITLE
Tweaked gun vendor prices

### PR DESCRIPTION
**Pull request by Jackofashepard**
## About The Pull Request

I reduced almost all the prices in the gun vendors with items that relate to guns (guns, magazines and ammo) as almost all of these are early to mid tier items and have fallen into disuse. Many people regard these prices as not worth it as they often find better guns and ammo in maint a lot faster than they attain the credits to buy these.

## Why It's Good For The Game

In short this is a effort to make these weapons see a bit more use and be more affordable for the average eris role. Open to more suggestions on the prices as i more or less just willy nilly reduced them by 200-600 creds except for special cases.

## Changelog
:cl:
tweak: reduced gun vendor prices (and prices on guns themselves).
/:cl:

### CODE REVIEW

**Review by Iglpigl — CHANGES_REQUESTED** [review]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Jackofashepard** [comment]
While i would love to take you seriously you have exactly 0 actual contributions aside from arguing and saying "conflicts". I am aware this will affect how often these spawn but it is not an issue as these are low to midtier guns anyways, if they spawn more often that is great as it will make less high tier guns spawn more often which has been a problem lately. On your other point i already stated why this PR was made, if you disagree its understandable but you should try constructive criticism instead of "ree ree no reason to make this ree". Try some of these and come again, thank you.

**Comment by Iglpigl** [comment]
> While i would love to take you seriously you have exactly 0 actual contributions aside from arguing and saying "conflicts". I am aware this will affect how often these spawn but it is not an issue as these are low to midtier guns anyways, if they spawn more often that is great as it will make less high tier guns spawn more often which has been a problem lately. On your other point i already stated why this PR was made, if you disagree its understandable but you should try constructive criticism instead of "ree ree no reason to make this ree". Try some of these and come again, thank you.

I would love to take you seriously, but now for sure I know this was done on a whim, both Novakovic and Gladstone are definitely not low-mid tear guns. Now the loot sistem, yes it will work how you think but not entirely, for example if any of guns that changed prices were to spawn yes they will be more frequent, but not only in things that have guns as loot but all loottables, it's a pretty big change, the reasoning you gave for the PR is incredibly flimsy and we have for years been trying to make good guns more rare and decent and bad guns more common, this is making both good and decent and bad guns more common which isn't wanted nor needed.
Now considering you ignored contributing guidelines I will unfortunately have to spell it out to you: "Don't nerf/buff values of things SIX TIMES in one go. Nerf values for 35-50%, see the results on live server, note them, nerf further when there is still a problem. "
Changing 300 to 100 is definitely more then 50%.
I get that you want people to have guns but don't give them almost the best stuff in the game roundstart, they aren't antags nor IH.

**Comment by Jackofashepard** [comment]
I am still failing to see a downside. Having more decent guns spawn is perfectly fine as currently we see a lot more cases where dirt tier and top tier guns rampantly spawning, i promise you vagabonds and other roles find way way better guns than all of these combined even 25 minutes into rounds. Also the guidelines are guidelines, not laws.

**Comment by Iglpigl** [comment]
> I am still failing to see a downside. Having more decent guns spawn is perfectly fine as currently we see a lot more cases where dirt tier and top tier guns rampantly spawning, i promise you vagabonds and other roles find way way better guns than all of these combined even 25 minutes into rounds. Also the guidelines are guidelines, not laws.

Well currently dirt tier guns aren't spawning enough, gun parts are extremely rare, and we do not want top tier guns rampantly spawning at all. About guidelines your point is valid, thing about vagabonds yep, but other roles hell no unless you want to encourage departments to junk, technomancers and medical (without science) for example can't make any guns on their own and don't junk, by changing vendor prices they can get the best possible stuff for them(without junking which we are trying to discourage for more then a year and still failing) much faster then in past.

**Comment by MLGTASTICa** [comment]
I believe that the vendor guns are already very decent , most of the stuff you find in maintenance is MEH at most.
I believe guns should stay expensive to keep people caring about having them , if we make them super cheap people will stop caring about guns and the cost behind them.

**Comment by TheShown911** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by calvinbeall** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Wouju** [comment]
Firstly th're is absolutely nay reasoneth f'r this to beest madeth, secondly 'twill screweth up loot and maketh all these items m're ingraft, thirdly, one and possibly m're items price changes art directeth violation to contributing guidelines, thee shouldst tryeth reading those folk. 
 https://wiki. cev-'ris. com/content_guidelinesen

**Comment by Feldnost-ui** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by CeUvi** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by SaltyOldBull** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Alliostra** [comment]
3 similar comments.

**Comment by TheShown911** [comment]
They are...

**Comment by drexample** [comment]
This seems to violate the content guidelines, buddy. I think you should try reading th-... Oh why bother, I'll just change the guidelines to allow it lmao.

**Comment by hyperioo** [comment]
Firstly there is absolutely no😣 reason for this to be made. Secondly it🇮🇹 will screw up🆙 loot and make all these items ➕more➕ common. Thirdly, 1️⃣one 1️⃣ and possibly more➕ items price changes🚼 are direct violation to contributing guidelines, you should try 📙reading 📙 them.

**Comment by Omni44** [comment]
Put me in the screencap.

**Comment by Tokanova** [comment]
Violate those content guidelines.
Just fuck em up.
Ram this PR through.

File: code/modules/projectiles/guns/projectile/boltgun.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	fire_sound = 'sound/weapons/guns/fire/sniper_fire.ogg'
 	reload_sound = 'sound/weapons/guns/interact/rifle_load.ogg'
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_PLASTIC = 10)
-	price_tag = 1600
+	price_tag = 900
```
Comment (TheShown911) [inline code/modules/projectiles/guns/projectile/boltgun.dm:25]: 1000

**Comment by Medcar** [comment]
Real chads don't read guidelines

**Comment by Jackofashepard** [comment]
To be very honest, i forgot we had any

**Review by Firefox13 — APPROVED** [review]
